plop-templates/component.tsx.hbs

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```

Comment:
I don't know that we need another template file.  I think we can do this by altering the existing template file.

Reply:
Yes I was able to do this and it will significantly reduce the code footprint.

Reply:
Done!